Bottom: ee24f3c472a215459d5609e13fc6e4b669cedb96
Top:    557ac88778798155ac8936fb9d3bdfb005c41a02
Author: Jannis Leidel <jannis@leidel.info>
Date:   2019-05-13 23:23:32 +0200

Fix migration names.


---
diff --git a/migrations/versions/280daa582976_.py b/migrations/versions/280daa582976_.py
index 6bf3b6bd..578ba29a 100644
--- a/migrations/versions/280daa582976_.py
+++ b/migrations/versions/280daa582976_.py
@@ -1,7 +1,7 @@
 """Add column metadata and table metadata
 
 Revision ID: 280daa582976
-Revises: b8a479422596
+Revises: 151a4c333e96
 Create Date: 2019-01-24 18:23:53.040608
 
 """
@@ -11,7 +11,7 @@ import sqlalchemy as sa
 
 # revision identifiers, used by Alembic.
 revision = '280daa582976'
-down_revision = 'b8a479422596'
+down_revision = '151a4c333e96'
 branch_labels = None
 depends_on = None
